src/main/scala/com/hortonworks/spark/sql/hive/llap/HS2JDBCWrapper.scala

```diff
@@ -174,7 +174,7 @@ class JDBCWrapper {
   def columnString(dataType: DataType, dataSize: Option[Long]): String = dataType match {
     case IntegerType => "INTEGER"
     case LongType => "BIGINT"
-    case DoubleType => "DOUBLE PRECISION"
```

Comment:
This should return Spark types. `DOUBLE PRECISION` is officially used in Hive since HIve 2.2.0.